atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTestTools.java

```diff
@@ -0,0 +1,116 @@
+import org.apache.cassandra.thrift.CqlRow;
+import org.apache.thrift.TException;
+
+import com.google.common.collect.Iterables;
+import com.google.common.primitives.Longs;
+
+public final class SchemaMutationLockTestTools {
+
+    private final CassandraClientPool clientPool;
+    private final UniqueSchemaMutationLockTable lockTable;
+
+    public SchemaMutationLockTestTools(CassandraClientPool clientPool, UniqueSchemaMutationLockTable lockTable) {
+        this.clientPool = clientPool;
+        this.lockTable = lockTable;
+    }
+
+    // for handling old non-heartbeat lock values
+    public CqlResult setLocksTableValue(long lockId) throws TException {
+        String lockValue = CassandraKeyValueServices.encodeAsHex(Longs.toByteArray(lockId));
+        return setLocksTableValueInternal(lockValue);
+    }
+
+    public CqlResult setLocksTableValue(long lockId, int heartbeatCount) throws TException {
+        String lockValue = getHexEncodedBytes(SchemaMutationLock.lockValueFromIdAndHeartbeat(lockId, heartbeatCount));
+        return setLocksTableValueInternal(lockValue);
+    }
+
+    public CqlResult truncateLocksTable()
+            throws TException {
+        return clientPool.run(client -> {
+            String truncateCql = String.format("TRUNCATE \"%s\";", lockTable.getOnlyTable().getQualifiedName());
+            return runCqlQuery(truncateCql, client, ConsistencyLevel.ALL);
+        });
+    }
+
+    public CqlResult readLocksTable() throws TException {
+        return clientPool.run(client -> {
+            String selectCql = "SELECT \"value\" FROM \"%s\" WHERE key = %s AND column1 = %s AND column2 = -1;";
+            String lockRowName = getHexEncodedBytes(CassandraConstants.GLOBAL_DDL_LOCK_ROW_NAME);
+            String lockColName = getHexEncodedBytes(CassandraConstants.GLOBAL_DDL_LOCK_COLUMN_NAME);
+            selectCql = String.format(selectCql, lockTable.getOnlyTable().getQualifiedName(), lockRowName, lockColName);
+            return runCqlQuery(selectCql, client, ConsistencyLevel.LOCAL_QUORUM);
+        });
+    }
+
+    public long readLockIdFromLocksTable() throws TException {
+        CqlResult result = readLocksTable();
+        Column resultColumn = getColumnFromCqlResult(result);
+        return SchemaMutationLock.getLockIdFromColumn(resultColumn);
+    }
+
+    public int readHeartbeatCountFromLocksTable() throws TException {
+        CqlResult result = readLocksTable();
+        Column resultColumn = getColumnFromCqlResult(result);
+        return SchemaMutationLock.getHeartbeatCountFromColumn(resultColumn);
+    }
+
+    private String getHexEncodedBytes(String str) {
+        return CassandraKeyValueServices.encodeAsHex(str.getBytes(StandardCharsets.UTF_8));
+    }
+
+    private CqlResult setLocksTableValueInternal(String hexLockValue) throws TException {
+        return clientPool.run(client -> {
+            String updateCql = "UPDATE \"%s\" SET value = %s WHERE key = %s AND column1 = %s AND column2 = -1;";
```

Comment:
You should inline this. It's bad practice to use a variable for the first parameter of `String.format` (or anything that has format parameters)